### DISCUSSION TITLE
adnauseum addon icon doesn't stay

### DISCUSSION THREAD

**Discussion by astro46** [dhowe/AdNauseam#2481] [Q&A]
I am using vivaldi, a chromium based browser.  The problem that I have been having for some time is that adnauseum is the only icon, in the taskbar list of addons, that won't stay where I place it. I open vivialdi and find that has shifted itself over to the left, no. 1 position.   I move it over to the right, and some time later, there it is again , on the left.  I know it is an important addon, but I would still like it to stay put.  It doesn't need to be no. 1.

What is going on?

**Comment by dhowe** [comment]
see https://github.com/dhowe/AdNauseam/issues/2482
